packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/selectors.js

```diff
@@ -0,0 +1,45 @@
+import isEqual from 'lodash/isEqual';
+import { createSelector } from 'reselect';
+
+import { REDUX_NAMESPACE } from './constants';
+import { initialState } from './reducer';
+
+// TODO: this is bad
+export const selectNamespace = (state) =>
+  state[REDUX_NAMESPACE] ?? state?.['0'][REDUX_NAMESPACE] ?? initialState;
```

Comment:
@joshuaellis This is where I need your help:

This should simply be

```js
export const selectNamespace = (state) =>
  state[REDUX_NAMESPACE] ?? initialState;
```

but I can't test `Stage` or `WorkflowAttributes` in isolation anymore, because `useInjectReducer` is only called in `EditView` for example, which doesn't wrap these components here for simplicity.

Redux complains I can't setup the namespace for the feature in the initial state directly:

https://github.com/strapi/strapi/blob/bc2d42374a663183ebd6542000a3c467a91f119a/packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/components/WorkflowAttributes/tests/WorkflowAttributes.test.js#L90-L114

but I have to nest it under `0`. Do you have any idea how I could work around that?

Reply:
I take look

Reply:
Nevermind. I've had an idea yesterday - the problem is using our custom `configureStore` helper. For these tests `createStore` is fine and fixes the issue.